content/blog/2017/10/linux-patching-management-with-ansible.md

```diff
@@ -0,0 +1,524 @@
++++
+author = "Brian Alcorn"
+author_url = "https://github.com/balcornc"
+categories = ["Brian Alcorn", "Encore"]
+date = "2017-10-31"
+description = "Centralized Linux Patching Activity Management for Consistency, Compliance, and Scalability"
+featured = ""
+featuredalt = ""
+featuredpath = ""
+linktitle = ""
+title = "Linux Patching Management With Ansible - Phase One"
+type = "post"
+
++++
+
+# Background
+
+While mulitple options exist for configuration and patching management, across multiple platforms, what we need is a vendor agnostic solution which, once built, will be easy to maintain, update, and enhance - and that won't require significant training for new associates to begin using.  For RHEL, we use Satellite to manage repositories, host collections, licensing, and some standard configs.  
+
+For ensuring hosts have patches successfully applied inside a tight timeline, Satellite provides feedback not unlike the steering wheel of a '78 Chevrolet pick-up - you're in the ballpark, but never really 100% sure.  For consistency and compliance, we need something with better real time feedback.
+
+In addition to "mushy" patch deployment feedback, Satellite is limited in the scope of automation it will accommodate.  We need a solution that can eventually  be turned loose to manage all patching activities on a schedule, without additional human input.  Ideally, once hosts, roles, playbooks and variables are fleshed out, it's simply a matter of defining and implementing a schedule to govern patch management.  Human interaction will only be necessary for adding/removing hosts, or when Ansible reports an error.
+
```

Comment:
After reading this intro i'm a little confused. You talk about satellite and only mention ansible in the last sentence.

I'd re-do this a little and say "here's what we were doing previously (satellite and its drawbacks).... and here's what we're trying to achieve (full automation)... then, here's how we're going to achieve it (ansible)"